jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/ResponseTest.java

```diff
@@ -56,6 +59,95 @@ public void dispose() throws Exception
         connector = null;
     }
 
+    @Test
+    public void testGET() throws Exception
+    {
+        server.setHandler(new Handler.Abstract()
+        {
+            @Override
+            public boolean handle(Request request, Response response, Callback callback)
+            {
+                callback.succeeded();
+                return true;
+            }
+        });
+        server.start();
+
+        String request = """
+                GET /path HTTP/1.0\r
+                Host: hostname\r
+                \r
+                """;
+        HttpTester.Response response = HttpTester.parseResponse(connector.getResponse(request));
+        assertEquals(HttpStatus.OK_200, response.getStatus());
+    }
```

Comment:
Not sure what this is testing that is not already tested many other times?

Reply:
It is the simplest test case within the unit test. Something that can be used to see how a simple response works and also something that other tests can be based on.   Too often when I'm writing a new test case, I find it hard to find a simple test without complications to use as the basis for another test.   Hence I now try to always have a really simple test in every unit test as a basis.